tools/dependencies/known-dependencies.txt

```diff
@@ -644,6 +645,7 @@ snakeyaml-1.24.jar
 snappy-0.3.jar
 snappy-java-1.1.4.jar
 snappy-java-1.1.7.1.jar
+snappy-java-1.1.7.3.jar
```

Comment:
Please consider if it is possible to unify the version, which we can manage in maven dependency management.

Reply:
It looks the dependency include the snappy inside
```xml
<dependency>
            <groupId>org.apache.flink</groupId>
            <artifactId>flink-connector-kafka_2.11</artifactId>
            <version>${flink.version}</version>
        </dependency>
```
use `mvn dependency:tree`
```text
[INFO] org.apache.seatunnel:flink-sql-connector-kafka:jar:2.1.1-SNAPSHOT
[INFO] +- org.apache.flink:flink-connector-kafka_2.11:jar:1.13.6:compile
[INFO] |  +- org.apache.kafka:kafka-clients:jar:2.4.1:compile
[INFO] |  |  +- com.github.luben:zstd-jni:jar:1.4.3-1:compile
[INFO] |  |  +- org.lz4:lz4-java:jar:1.6.0:compile
[INFO] |  |  +- org.xerial.snappy:snappy-java:jar:1.1.7.3:compile
[INFO] |  |  \- org.slf4j:slf4j-api:jar:1.7.28:compile
[INFO] |  +- org.apache.flink:flink-connector-base:jar:1.13.6:compile
[INFO] |  \- org.apache.flink:force-shading:jar:1.13.6:compile
```

I'm not sure if that's a good idea to exclude the snappy(1.1.7.3) from `flink-connector-kafka_2.11` and add 1.1.7.1 into module dependency.

Do you have any ideas ？ @CalvinKirs

Reply:
This is strange, this PR didn't add new dependency but import a subdendency. Anyway, I unify the version of `slf4j-api` and `snappy-java`.

Reply:
needs the test, but you can create a new PR after your test is done. (If updated can great work)

> It looks the dependency include the snappy inside
> 
> ```
> <dependency>
>             <groupId>org.apache.flink</groupId>
>             <artifactId>flink-connector-kafka_2.11</artifactId>
>             <version>${flink.version}</version>
>         </dependency>
> ```
> 
> use `mvn dependency:tree`
> 
> ```
> [INFO] org.apache.seatunnel:flink-sql-connector-kafka:jar:2.1.1-SNAPSHOT
> [INFO] +- org.apache.flink:flink-connector-kafka_2.11:jar:1.13.6:compile
> [INFO] |  +- org.apache.kafka:kafka-clients:jar:2.4.1:compile
> [INFO] |  |  +- com.github.luben:zstd-jni:jar:1.4.3-1:compile
> [INFO] |  |  +- org.lz4:lz4-java:jar:1.6.0:compile
> [INFO] |  |  +- org.xerial.snappy:snappy-java:jar:1.1.7.3:compile
> [INFO] |  |  \- org.slf4j:slf4j-api:jar:1.7.28:compile
> [INFO] |  +- org.apache.flink:flink-connector-base:jar:1.13.6:compile
> [INFO] |  \- org.apache.flink:force-shading:jar:1.13.6:compile
> ```
> 
> I'm not sure if that's a good idea to exclude the snappy(1.1.7.3) from `flink-connector-kafka_2.11` and add 1.1.7.1 into module dependency.
> 
> Do you have any ideas ？ @CalvinKirs
needs the test, but you can create a new PR after your test is done. (If updated can good to work)